package.json

```
@@ -8,10 +8,7 @@
   },
   "dependencies": {
     "del": "^5.1.0",
-    "digitalmarketplace-frontend-toolkit": "https://github.com/alphagov/digitalmarketplace-frontend-toolkit.git#v36.1.0",
-    "digitalmarketplace-govuk-frontend": "^0.9.1",
-    "govuk-elements-sass": "3.0.3",
-    "govuk_frontend_toolkit": "5.0.3",
+    "digitalmarketplace-govuk-frontend": "github:alphagov/digitalmarketplace-govuk-frontend#26080ba",
```

```suggestion
    "digitalmarketplace-govuk-frontend": "github:alphagov/digitalmarketplace-govuk-frontend#v1.0.0",
```
Can we point to tag rather than commit hash?

---

I'm happy to - we should update dm-govuk-frontend's `release-to-branch` terminal output to reflect that though : https://github.com/alphagov/digitalmarketplace-govuk-frontend/blob/1ffd3ce34e52a7ad6e49fb128b362b394ea11252/bin/release-to-branch.sh#L44

---

I think release to branch is more for testing but once released we should use a tagged release.

---

@gidsg I don't think we can do a release, because it's a long-running branch; we need to have changes applied to digitalmarketplace-govuk-frontend to support govuk-frontend v3, and I don't think we want those changes in the default branch.